docs/content/developer/apk-distribution.md

```diff
@@ -3,4 +3,20 @@ The web app [APK distribution](https://apps.dhis2.org/app/dff273fc-909e-48af-b15
 
 This parameter is automatically used by the official Android Capture app, which will offer the update when a new version is identified. Other custom applications might use this parameter to control their own updates. This logic must be implemented at application level.
 
+The APK distribution app allows the definition of different versions by user group. The SDK will internally handle this logic and determine the version that must be used by the current user.
+
+The app can get this version by using the LatestAppVersion repository:
+
+```kt
+d2.settingModule().latestAppVersion().get()
+```
+
+This version is updated on every metadata sync. If you want to check for updates without triggering a whole metadata sync, you can use this method:
```

Comment:
This version is updated with each metadata sync. To check for updates without triggering a full metadata sync, this method can be used: